### PR TITLE
Release 3.0.0 to Release 3.1.0

### PR DESCRIPTION
**Pull request by keshavprasadms**

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/search-service_application.conf [hunk 1]
```diff
@@ -344,6 +344,6 @@ language.map={"Hindi":"hi", "English":"en", "Telugu":"te", "Kannada":"ka", "Tami
 channel.default="in.ekstep"
 compositesearch.index.name="{{ compositesearch_index_name }}"
 
-content.tagging.backward_enable=true
+content.tagging.backward_enable=false
```
Comment (SMYALTAMASH) [inline ansible/roles/stack-sunbird/templates/search-service_application.conf:347]: This need to be variablized and not hardcoded, will ask the team to do it in another PR.

Comment (keshavprasadms) [reply ansible/roles/stack-sunbird/templates/search-service_application.conf:347]: As this already gone libe, it needs to be done as a hotfix / in 3.1
